### ISSUE TITLE
no ClojureScript tests exits with non-zero

### ISSUE THREAD

**Issue by bostonaholic** [emezeske/lein-cljsbuild#356]
In lein-cljsbuild version `1.0.4` if there are no ClojureScript tests, `lein test` will exit with status 1. I do not believe this is the correct behavior. A CI build will now fail when upgrading to lein-cljsbuild 1.0.4 that has no ClojureScript tests.

```
$ lein test
Compiling ClojureScript.

lein test user

Ran 0 tests containing 0 assertions.
0 failures, 0 errors.
Could not locate test command .
$ echo $?
1
$
```

Current workaround is to write a single trivially passing ClojureScript test.

Another workaround is to add `:cljsbuild {:test-commands {"dummy" ["true"]}}` to `project.clj`.

**Comment by bostonaholic** [comment]
Sorry for the commit confusion. Accidentally committed the wrong changes. 12a2b13 should be the correct change.

**Comment by cemerick** [comment]
12a2b13 applied to the `develop/1.x` branch, and available in `1.0.5-SNAPSHOT`, thanks!
